app/models/manageiq/providers/base_manager/event_catcher/runner.rb

```diff
@@ -89,23 +88,8 @@ def before_exit(message, _exit_code)
     end
   end
 
-  attr_reader :filtered_events
-
-  # Called when there is any change in BlacklistedEvent
-  def sync_blacklisted_events
-    return unless @ems
-
-    filters = @ems.blacklisted_event_names
-
-    if @filtered_events.nil? || @filtered_events != filters
-      adds    = filters - @filtered_events
-      deletes = @filtered_events - filters
-
-      @filtered_events = filters
-      _log.info("Synchronizing blacklisted events: #{filters}")
-      _log.info("   Blacklisted events added: #{adds}")
-      _log.info("   Blacklisted events deleted: #{deletes}")
-    end
+  def filtered_events
+    @ems.class.default_blacklisted_event_names
   end
```

Comment:
NOTE we probably don't even need this method since the providers could use their Settings.ems.ems_type directly